### ISSUE TITLE
Attribute Name Incorrectly Marked as Incorrect.

### ISSUE THREAD

**Issue by iDad5** [validator/validator#1522]
URL being validated or code to reproduce error:
```
<input type="checkbox" id="sov1-o1-q1-p4" value="1" data-value-type="sub-value" data-linked⁺group-exclusion="lärm">
```
paste error messages into code block
```
Warning: Attribute data-linked⁺group-exclusion is not serializable as XML 1.0.
```

I'm using the Superscript Plus Sign in my data-attribute to reduce the likelihood of conflicts. Its codepoint 207A is allowed and valid in HTML5 data-attributes according to all my research. Or am I mistaken?

**Comment by sideshowbarker** [comment]
Running that through an XML parser, I get this:

> error fatal: Not a name character, U+207a

So yeah, according to an XML parser (and thus apparently according to the XML spec), that Superscript Plus Sign is not an allowed name character.

But when that’s in an attribute name in HTML document, it’s not a fatal error, but instead just a warning.

And the reason there’s even a warning — rather than no message at all — is that what it’s trying to tell you is: _If_ you try to take that document and serve with an XML media type (aka, as an “XHTML” document), it’s not well-formed XML, and XML parsers will fail to parse it.
